dvc/command/plots.py

```diff
@@ -32,21 +32,12 @@ def _func(self, *args, **kwargs):
         raise NotImplementedError
 
     def run(self):
-        fields = None
-        jsonpath = None
-        if self.args.select:
-            if self.args.select.startswith("$"):
-                jsonpath = self.args.select
-            else:
-                fields = set(self.args.select.split(","))
         try:
             plots = self._func(
                 targets=self.args.targets,
                 template=self.args.template,
-                fields=fields,
```

Comment:
Not removing the API part because it is covered by the tests and we will definitely end up using it in the nearest future. Just removing it from CLI so one tries to use it in 1.0.